src/data/load_geometry.js

```diff
@@ -49,7 +49,7 @@ export default function loadGeometry(feature: FeatureWithGeometry, canonical?: C
     const extent = feature.extent;
     const extentScale = EXTENT / extent;
 
-    if (canonical && tileTransform && tileTransform.projection.name !== 'mercator') {
+    if (canonical && tileTransform && tileTransform.projection.isReprojectedInTileSpace) {
```

Comment:
Re-projection of points and other shapes used by symbol layers is done at runtime. I'm wondering if this check could interfere with symbol layers because the re-projection can't be done in more granular level?

Reply:
I don't think so, if I understand the question correctly.

`isReprojectedInTileSpace` is introduced to have a clear extension of `!= 'mercator'` to account for globe. With the introduction of non-mercator, for every check in the codebase where we have `== 'mercator'` we have to extend it to account for globe, so it's now an explicit boolean as part of the projection interface instead of:
```
if (canonical && tileTransform && tileTransform.projection.name !== 'mercator' && tileTransform.projection.name !== 'globe')
```
It should never fall through this block as `isReprojectedInTileSpace` is always false for globe, even if we have symbol/shapes runtime re-projection. Maybe there's a better name, as we have several concepts of re-projection now:
- globe symbol/shape runtime re-projections 
- non-mercator tile space re-projections, under `isReprojectedInTileSpace`